### PR TITLE
OCPBUGS-22072: Disable the http2 endpoint and bump the net package

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -98,10 +101,10 @@ func main() {
 	namespaceSelector := cache.ByObject{
 		Field: fields.ParseSelectorOrDie(fmt.Sprintf("metadata.namespace=%s", operatorNamespace)),
 	}
+
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:             scheme,
 		MetricsBindAddress: *metricsAddr,
```
Comment (msherif1234) [inline main.go:107]: @fedepaol I assume this version of controller-runtime doesn't allow you to do the same for metrics ?

Comment (fedepaol) [reply main.go:107]: right, plus the default value for metrics is :0 (meaning, we don't listen to metrics)

**Comment by fedepaol** [comment]
Flake due to pod not starting
/override ci/prow/metallb-e2e-metal-ipi-ovn

**Pull request merged**